circle.yml

```diff
@@ -4,19 +4,19 @@ machine:
 
 dependencies:
     pre:
-        - curl -L https://github.com/commercialhaskell/stack/releases/download/v1.3.0/stack-1.3.0-linux-x86_64.tar.gz | tar zx -C /tmp
-        - sudo mv /tmp/stack-1.3.0-linux-x86_64/stack /usr/bin
+        - curl -L https://github.com/commercialhaskell/stack/releases/download/v1.6.5/stack-1.6.5-linux-x86_64.tar.gz | tar zx -C /tmp
+        - sudo mv /tmp/stack-1.6.5-linux-x86_64/stack /usr/bin
     override:
-        - stack build --stack-yaml=shell/stack.yaml --only-dependencies --fast --install-ghc --test --ghc-options="-O0"
+        - /usr/bin/stack build --stack-yaml stack.yaml --only-dependencies --fast --install-ghc --test --ghc-options="-O0"
```

Comment:
/usr/bin is in PATH, why specify it here?

`stack.yaml` in current directory does not need to be specified in an argument